src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -68,6 +69,19 @@ public void setTisUrl(final String tisUrl) {
         this.tisUrl = tisUrl;
     }
 
+    /**
+     * Gets the callback url for the Test Integration System
+     *
+     * @return the TIS callback URL
+     */
+    public String getTisCallbackUrl() {
```

Comment:
Ah shoot, I totally spaced on that.  From an architectural standpoint, as the sender, this service should be handling the callback.  Even if it just forwards the response back to the TDS_ExamService (maybe via additional messaging.)